### PR TITLE
feature/FOUR-14385: STORY Process Catalog updates

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps

- The Figma includes a desig for the Searches in categories to clear the searh (Review Figma ¨Search Categories).
- The Figma includes a design for a result that may be was broke or something was wrong, the message will be “Something went wrong Please try again in a few moments, or contact your admin if the problem persists.” review Figma: “Browser Clarifications”.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-14385

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-686d56c9fb.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@luNunezProcessmaker please review the image
image

File: resources/js/processes-catalogue/components/utils/EmptySearch.vue [hunk 1]
```diff
@@ -1,24 +1,43 @@
 <template>
-  <div>
+  <div class="py-5">
     <div class="d-flex justify-content-center my-5">
-      <i class="empty-processes fa fa-umbrella-beach" />
+      <img
+          class="image d-flex empty-processes"
+          src="/img/launchpad-images/emptyResults.svg"
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/utils/EmptySearch.vue:6]: This image exist?

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-686d56c9fb.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/utils/EmptySearch.vue [hunk 2]
```diff
@@ -1,24 +1,43 @@
...
+          alt="Empty Results"
+        >
     </div>
-    <p class="text-center message-empty">
+    <p class="text-center title-empty">
       {{ $t("No Processes Available") }}
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/utils/EmptySearch.vue:11]: Please update this label according FIGMA:
`No results to show`
Please consider the translations too

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-686d56c9fb.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-686d56c9fb.engk8s.processmaker.net

**Pull request merged**